tutorials.md

```diff
@@ -39,6 +39,7 @@
 * [Colouring words by topic in a document, print words in a topics](https://github.com/RaRe-Technologies/gensim/blob/develop/docs/notebooks/topic_methods.ipynb)
 * [Topic Coherence, a metric that correlates that human judgement on topic quality.](https://github.com/RaRe-Technologies/gensim/blob/develop/docs/notebooks/topic_coherence_tutorial.ipynb) 
 * [Compare topics and documents using Jaccard, Kullback-Leibler and Hellinger similarities] (https://github.com/RaRe-Technologies/gensim/blob/develop/docs/notebooks/similarity_metrics.ipynb)
+* [America's Next Topic Model slides](https://speakerdeck.com/tmylk/americas-next-topic-model?slide=6)
```

Comment:
Pleases add more context -- what are the slides about, why should the reader click on this link? What will they learn?